less/data_selection/matching.py

```diff
@@ -44,23 +36,26 @@ def calculate_influence_score(training_info: torch.Tensor, validation_info: torc
         training_info, validation_info.transpose(0, 1))
     return influence_scores
 
+checkpoints = [int(i) for i in args.ckpts.split(" ")]
 
+checkpoint_weights = [float(i) for i in args.checkpoint_weights.split(" ")]
 # renormalize the checkpoint weights
```

Comment:
I'm not sure how this matches with line 14, can you please explain?

Reply:
It was something else (I wasn't sure about checkpoint normalization, but misclicked at the line number), but after looking once again at the code I figured out the answer, so nevermind :)